module/importer-magento-1/src/Infrastructure/Processor/Step/Magento1CategoryProcessor.php

```diff
@@ -32,74 +31,61 @@ class Magento1CategoryProcessor implements Magento1ProcessorStepInterface
      */
     private CommandBusInterface $commandBus;
 
+    /**
+     * @var string[]
+     */
+    private array $categories;
+
     /**
      * @param CommandBusInterface $commandBus
      */
     public function __construct(CommandBusInterface $commandBus)
     {
         $this->commandBus = $commandBus;
+        $this->categories = [];
     }
 
     /**
      * @param Import            $import
-     * @param array             $products
+     * @param ProductModel      $product
      * @param Transformer       $transformer
      * @param Magento1CsvSource $source
-     * @param Progress          $steps
-     *
-     * @return int
      */
     public function process(
         Import $import,
-        array $products,
+        ProductModel $product,
         Transformer $transformer,
-        Magento1CsvSource $source,
-        Progress $steps
-    ): int {
-        $result = [];
-        foreach ($products as $sku => $product) {
-            $default = $product->get('default');
+        Magento1CsvSource $source
+    ): void {
+        $default = $product->get('default');
+        if (array_key_exists('esa_categories', $default) && $default['esa_categories'] !== '') {
```

Comment:
`!empty($default['esa_categories'])`

Reply:
Empty work also on false and 0 and false

Reply:
But what if `esa_categories` have one of values : `false`, `0`, `null` or `"false"`?